lib/node-repo.js

```diff
@@ -137,15 +137,50 @@ function fetchLabelPages (options, startPageNum, cb) {
   })
 }
 
+function getBotPrLabels (options, cb) {
```

Comment:
Nice 👍 This might be handy for subsystem labels as well, if we were to react on PR updates for those labels as well.